### PR TITLE
{math}[intel/2018a] Keras v2.2.4, Theano v1.0.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2060.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f366fb36384d7dc9930bc71251b805b1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2417.golett.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3341e4c026afd1b21f726b965039c3b2 for a full test report.

File: easybuild/easyconfigs/k/Keras/Keras-2.2.4-intel-2018a-Python-3.6.4.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        'source_urls': ['https://pypi.python.org/packages/source/K/Keras-Applications'],
+        'source_tmpl': 'Keras_Applications-%(version)s.tar.gz',
+        'checksums': ['a03af60ddc9c5afdae4d5c9a8dd4ca857550e0b793733a5072e0725829b87017'],
+        'modulename': 'keras_applications',
+    }),
+    ('Keras-Preprocessing', '1.0.5', {
+        'source_urls': ['https://pypi.python.org/packages/source/K/Keras-Preprocessing'],
+        'source_tmpl': 'Keras_Preprocessing-%(version)s.tar.gz',
+        'checksums': ['ef2e482c4336fcf7180244d06f4374939099daa3183816e82aee7755af35b754'],
+        'modulename': 'keras_preprocessing',
```
Comment (ocaisa) [inline easybuild/easyconfigs/k/Keras/Keras-2.2.4-intel-2018a-Python-3.6.4.eb]: Ditto

File: easybuild/easyconfigs/k/Keras/Keras-2.2.4-intel-2018a-Python-3.6.4.eb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+use_pip = True
+
+exts_list = [
+    ('Keras-Applications', '1.0.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/K/Keras-Applications'],
+        'source_tmpl': 'Keras_Applications-%(version)s.tar.gz',
+        'checksums': ['a03af60ddc9c5afdae4d5c9a8dd4ca857550e0b793733a5072e0725829b87017'],
+        'modulename': 'keras_applications',
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/k/Keras/Keras-2.2.4-intel-2018a-Python-3.6.4.eb]: Indeed, thanks to https://github.com/easybuilders/easybuild-easyblocks/pull/1560 this is no longer needed, I'll clean it up.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2417.golett.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/041d2066122d1f98b57685e4141cbfa6 for a full test report.

File: easybuild/easyconfigs/k/Keras/Keras-2.2.4-intel-2018a-Python-3.6.4.eb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        'source_tmpl': 'Keras_Preprocessing-%(version)s.tar.gz',
+        'checksums': ['ef2e482c4336fcf7180244d06f4374939099daa3183816e82aee7755af35b754'],
+    }),
+    (name, version, {
+        'source_urls': ['https://pypi.python.org/packages/source/K/Keras'],
+        'checksums': ['90b610a3dbbf6d257b20a079eba3fdf2eed2158f64066a7c6f7227023fd60bc9'],
+    }),
+]
+
+sanity_check_paths = {
```
Comment (ocaisa) [inline easybuild/easyconfigs/k/Keras/Keras-2.2.4-intel-2018a-Python-3.6.4.eb]: Isn't this particular sanity check also moot with the new `PythonBundle`?

Comment (boegel) [reply easybuild/easyconfigs/k/Keras/Keras-2.2.4-intel-2018a-Python-3.6.4.eb]: Ah, right, yes it is, this is exactly the default used by `PythonPackage`... Cleaning up...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
gligar03.gligar.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/461be0f9f8325faf9c61599b4ac8f53a for a full test report.

**Pull request merged**